app/routes/tasks.js

```diff
@@ -0,0 +1,57 @@
+import {computed} from '@ember/object';
+import Route from '@ember/routing/route';
+import {inject} from '@ember/service';
+
+export default Route.extend({
+  // services
+  github: inject(),
+  organizations: inject(),
+  store: inject(),
+
+  queryParams: {
+    org: {
+      refreshModel: true,
+    },
+  },
+
+  currentOrg: computed('params.org', function () {
+    return this.get('organizations').fetch(this.get('params')['org']);
+  }),
+
+  model(params) {
+    this.set('params', params);
+    let org = this.get('organizations').fetch(params['org']);
+    let githubParams = [];
+    let store = this.get('store');
```

Comment:
In terms of ordering your variable declarations here, I'd group those using `this.get` together, especially when that `githubParams` doesn't really logically group with either of them (though if you'd like to keep it with `org`, I'd just move `store` above both of them). This'll just make it a little more visually appealing and cleaner to look at.

Reply:
Good catch. Will update.